```
======================================================================
FAIL: test_dump (tests.test_performance.TestPerformance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/private/tmp/nix-build-python3.9-jsons-1.6.3.drv-0/source/tests/test_performance.py", line 45, in test_dump
    self._do_test_dump(16)
  File "/private/tmp/nix-build-python3.9-jsons-1.6.3.drv-0/source/tests/test_performance.py", line 67, in _do_test_dump
    self.assertTrue(delta_sec3 < time_limit, 'The operation took {} seconds'.format(delta_sec3))
AssertionError: False is not true : The operation took 24.88201 seconds
```

Suggest trying to disable any timing-related tests. Better not to have builds failing because they're running on a heavily loaded machine. `unittest` doesn't have an easy cli-based test-skipping mechanism so usually the best way is to `substituteInPlace` the source and change e.g. `test_dump` into `dont_test_dump` so `unittest` doesn't "discover" it.

--- a/pkgs/development/python-modules/jsons/default.nix
+++ b/pkgs/development/python-modules/jsons/default.nix
@@ -1,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, attrs
+, pytestCheckHook
+, typish
+, tzdata
+}:
+
+buildPythonPackage rec {
+  pname = "jsons";
+  version = "1.6.3";
+
+  src = fetchFromGitHub {
+    owner = "ramonhagenaars";
+    repo = "jsons";
+    rev = "v${version}";
+    sha256 = "0sdwc57f3lwzhbcapjdbay9f8rn65rlspxa67a2i5apcgg403qpc";
+  };
+
+  propagatedBuildInputs = [
+    typish
+  ];
+
+  checkInputs = [
+    attrs
+    pytestCheckHook
+    tzdata
+  ];
+
+  disabledTestPaths = [
+    # These tests are based on timings, which fail
+    # on slow or overloaded machines.
+    "tests/test_performance.py"
+  ];
+
+  pythonImportsCheck = [
+    "jsons"
+  ];
+
+  meta = with lib; {
+    description = "Turn Python objects into dicts or json strings and back";
+    homepage = "https://github.com/ramonhagenaars/jsons";
+    license = licenses.mit;
+    maintainers = with maintainers; [ fmoda3 ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -4786,6 +4786,8 @@ in {
 
   jsonrpc-websocket = callPackage ../development/python-modules/jsonrpc-websocket { };
 
+  jsons = callPackage ../development/python-modules/jsons { };
+
   jsonschema = callPackage ../development/python-modules/jsonschema { };
 
   jsonstreams = callPackage ../development/python-modules/jsonstreams { };

